### ISSUE TITLE
Unix installer does not allow creating system links after launching a user-side install

### ISSUE THREAD

**Issue by gasche** [racket/distro-build#3]
When I install new software, I try to guess whether it will be located in the system part, in which case I use `sudo` to get administrator rights, or in the user part, in which case I do *not* use `sudo` to have the correct user rights for the installed files.

After looking at the list of options proposed by https://mirror.racket-lang.org/installers/6.5/racket-6.5-x86_64-linux.sh , I decided to install it in my $HOME with user rights only. But then, after unpacking is done, it offers me to add symlink in system parts, which sounds like a good idea: so I said "yes", but it failed as it did not have administrator rights.

This is fine and expected, but how do I fix the absence of system links now? I would like to be able to ask the installer: "you already unpacked at place FOO" (using the same place-selection tool I used on my first install), "please reuse this install but add system links". I tried `sudo sh ...-linux.sh --create-links /usr/local`, but the installer does not seem to be able to reuse an existing installation. I don't want to re-do the installation under `sudo`, as I'd rather keep the files in HOME with user rather than administrator permissions.

To summarize, I would like the following changes:

- An option to reuse an existing install (I'm fine with passing the installed location as a command-line parameter instead of having the menu selection again) instead of reinstalling one afresh
- In case the creation of system symlinks fail for insufficient rights, I would like an error message to be printed that gives the command-line to run (under administrator role) to re-create the symlinks for this install.
